front/src/modules/powerRestriction/helpers/__tests__/createPathStep.spec.ts

```diff
@@ -60,7 +60,7 @@ describe('findTrackOffset', () => {
       trackRanges
     );
 
-    expect(result).toEqual({ track: 'track_2', offset: 1050 });
+    expect(result).toEqual({ track: 'track_2', offset: 50 });
```

Comment:
You need to change the input, for the track_2, the begin must be 60 and the end 1060 (as we have seen that the begin < end in the backend response)

Reply:
Fixed :)